### ISSUE TITLE
Voron Trident and 2.4 not working with soft material

### ISSUE THREAD

**Issue by Ziiiwei** [VoronDesign/Voron-2#437]
I have built both with klicky probe and bed-atached z-endstop. Althought they work well with abs and all the other hard materials, they really sucked with soft TPU in my settings. The main issue is with the soft material geting streched and stuck in the PTFE tube. Both machine are driect drive design yet still heavily rely on long PTFE tube for the filment guidance. Pulling soft material through tube(especially with that 90 degree turn in back of printer) gonna create a lots of resistance, and as the material get streched and therefore thiner, the gear grip also decreace. Both machine are guaranteed to stuck after about 20 min run as the streching build up. Currently i have to open the top cover and put spool holder up in the top to get rid off the PTFE tube to make is print smoothly(the direct drive ender3 set up). Does anyone have similar issues and have already worked out a more elegant solution?
Another difficulty is I have to clean the z-endstop before the z-calibration when homing leaves sticky soft TPU on the stwich, otherwise I may have the print head diging into the bed. I'm currently trying switching to voron tap to solve this, but that will have potential conflicts with above tube less set up(as the print head move around pulling the fillament, if might just pull up the print head and trigger the switch). 
I'm guessing the way to actually solve this to have a separate drive at begining of the tube? But that really feels like a over kill, so anyone having this issue please help with my frustration...

**Comment by valadas** [comment]
What I do for tpu is that I leave the doors open and remove the tube. I mount the spool top front and just feed the toolhead direct without any tube.

**Comment by photodave** [comment]
I have had my V2.4 350mm for a little over a year and have printed several items with TPU from Polymaker, Duramic 3D and Eryone, with no failed prints with any of the TPU spools I tried. Im using the CW2 extruder and have my spool holder mounted on the outside rear right so the base of the spool is about 1" above the top of the skirts. I did mod the "bowden_retainer.stl" and extend it so when mounted above the spool, the bowden tube would line up roughly with the outside edge of the spool so it would also make a more gentle arc when going into the enclosure. Anyway I just thought I would throw that out there as I have not had any problems with various spools of TPU.
